views/email/test.md

```diff
@@ -0,0 +1 @@
+{{testID}}
```

Comment:
This test checks the validity of `composeEmail` in general not of data contribution notifications. That means it should pass whenever `composeEmail` does it's job instead of being dependent on a template that is subject to changes that may cause the test to break unnecessarily.